freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/FreeIpaClientFactory.java

```diff
@@ -211,15 +216,15 @@ private FreeIpaClientBuilder getFreeIpaClientBuilderForClusterProxy(Stack stack,
                 additionalHeadersStickySessionFirstRpc,
                 additionalHeadersStickySession,
                 stickyIdHeader,
-                CLUSTER_PROXY_ERROR_LISTENER);
+                CLUSTER_PROXY_ERROR_LISTENER, tracer);
```

Comment:
nit: all other parameters are each on their own lines.

Reply:
indeed, I'm fixing this